lib/dms.js

```diff
@@ -9,7 +9,7 @@ const logger = require('../utils/logger')
 class DmsModel {
   constructor(config) {
     this.config = config
-    this.api = config.get('API_URL')
+    this.api = config.get('INTERNAL_API_URL') || config.get('API_URL')
```

Comment:
So just to reflect your comment

> There are other places where frontend does connect to CKAN API (e.g., in custom themes developers don't always use this DMS library but they write custom code). But this DMS module makes 90% of requests, it also does a lot of concurrent requests where a package has lots of resources.


As I understand you still need API_URL to make regular request in some places. And that's cause custom themes are may also make request to CKAN_API, but why than needs to be done through NGINX, Is there a reason custom theme to not hit CKAN_API through internal network?